network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/take_action.html

```diff
@@ -29,7 +29,7 @@ <h4 class="take-action-title mb-4">{{ page.take_action_title }}</h4>
         <div class="bg-black p-4 dark-theme">
           <div
             class="join-us"
-            data-form-position="header"
+            data-form-position="body"
```

Comment:
This change turns the "Take action" callout from this:

![image](https://user-images.githubusercontent.com/177243/93262565-4c271080-f759-11ea-9eb0-cf4d27912864.png)

Into this:

![image](https://user-images.githubusercontent.com/177243/93262591-56490f00-f759-11ea-9c85-be1cdb02a352.png)